### DISCUSSION TITLE
GitHub-hosted macOS runners now offer Apple silicon (beta)

### DISCUSSION THREAD

**Discussion by irmo-de** [pawelsalawa/sqlitestudio#4775] [Ideas]
Hi,

since two weeks, there is a public beta for GitHub runners with Apple Silicon support.
https://github.com/github/roadmap/issues/528

Would be great to have SqliteStudio supporting them.

**Comment by tuffnatty** [comment]
I understand the linked issue as "We're postponing public M1 runners again, this time until the end of the year".

In the meanwhile, when MacPorts upgrades to Qt 5.15.9 or SQLiteStudio upgrades to Qt 6.x, whatever comes first, it will be possible to provide a universal x86_64/arm64 build.
